tap-gui/tls.md

```diff
@@ -0,0 +1,39 @@
+# Configuring TLS Certificate for Tanzu Application Platform GUI
+
+In order to configure a TLS certificate for Tanzu Application Platform GUI, follow the steps:
+
+1. Add a certificate to your cluster, in the `tap-gui` namespace. A simple example is provided.
+    
+    * Create a certificate.yaml file defining an Issuer and a Certificate, replacing `INGRESS-DOMAIN` with the value of your domain.
+
+        ```yaml
+        apiVersion: cert-manager.io/v1
+        kind: Issuer
+        metadata:
+        name: ca-issuer
+        namespace: tap-gui
+        spec:
+        selfSigned: {}
+        ---
+        apiVersion: cert-manager.io/v1
+        kind: Certificate
+        metadata:
+        name: tap-gui-cert
+        namespace: tap-gui
+        spec:
+        secretName: tap-gui-cert
+        dnsNames:
+        - tap-gui.INGRESS-DOMAIN
+        issuerRef:
+            name: ca-issuer
+        ```
+    * add the Issuer and Certificate to your cluster:
+
+        ```kubectl apply -f certificate.yaml```
+
+2. Update your `tap-gui` values to include a top level `tls` key, with subkeys for `namespace` and `secretName`. The namespace should refer to the namespace where tap-gui is running. The secret name should refer to the `secretName` value defined in your Certificate resource above.
```

Comment:
```suggestion
2. Update your `tap-gui` values to include a top level `tls` key, with subkeys for `namespace` and `secretName`. The namespace should refer to the namespace containing the above `Certificate` object. The secret name should refer to the `secretName` value defined in your Certificate resource above.
```
I may have misunderstood how the [TLSCertificateDelegation bit](https://gitlab.eng.vmware.com/project-star/pstar-backstage-poc/-/blob/develop/package-contents/config/ingress.yml) works, but I think you can put your `Certificate` in a different namespace than the one containing the tap-gui HTTPProxy and the association should still work.